test/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.FunctionalTests/DatabaseErrorPageMiddlewareTest.cs

```diff
@@ -77,8 +77,9 @@ public virtual Task Invoke(HttpContext context)
             }
         }
 
-        [ConditionalTheory]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [ConditionalFact(Skip = "aspnet/Diagnostics#350")]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
On EF, we use an external SQL Server, but it might be overkill to do here. Ultimately, these are just testing how our middleware intercepts exceptions, not the interaction with the database.